etna/datasets/tsdataset.py

```diff
@@ -416,10 +417,26 @@ def inverse_transform(self, transforms: Sequence["Transform"]):
         Applied in reversed order.
         """
         # TODO: return regressors after inverse_transform
```

Comment:
What about this TODO?

Reply:
Remove it, it is not actual anymore, we return the regressors inside the Transform

Reply:
Only feature selection return columns to the dataset, add test for this case

Reply:
We should fix this for feature selection transforms